Extensions/ExternalTfs/Src/Tasks/DownloadArtifactsTfsVersionControl/task.json

```diff
@@ -12,7 +12,7 @@
   "author": "ms-vscs-rm",  
   "version": {
     "Major": 15,
-    "Minor": 138,
+    "Minor": 139,
```

Comment:
Use current sprint version.

Reply:
Make this change for download build artifact task as well.

Reply:
done